Functions/Recursion/tests.py

```diff
@@ -5,6 +5,8 @@ def test_window():
     window = get_answer_placeholders()[0]
     if "factorial(n - 1)" in window and "*" in window:
         passed()
+    elif "factorial(n-1)" in window and "*" in window:
```

Comment:
We should probably add other options as well - ( n-1), (n-1 ), etc.
Or rewrite the test for good, as it is getting a bit awkward